planet/cli/subscriptions.py

```diff
@@ -263,20 +289,28 @@ async def list_subscription_results_cmd(ctx,
     help='Toolchain JSON. Can be a string, filename, or - for stdin.')
 @click.option(
     '--hosting',
-    type=types.JSON(),
-    help='Hosting JSON.  Can be a string, a filename, or - for stdin.')
+    default=None,
+    type=click.Choice([
+        "sentinel_hub",
+    ]),
+    help='Hosting configuration. Can be JSON, "sentinel_hub", or omitted.')
```

Comment:
Can this still be JSON if the type is `click.Choice(List[String,..])`?  The wording is a little bit different in the Subscriptions API implementation of this: https://github.com/planetlabs/planet-client-python/blob/28f2216e8fbaea5c0aa5104f4909d4be7982d978/planet/cli/orders.py#L297-L300.